src/endpoints/get-diffs.js

```diff
@@ -69,6 +55,6 @@ module.exports = cors(async (req, res) => {
     patch,
     changeLog,
     latestVersion: session.summary_version,
-    hashOfLatestState: hash(session.udt_json),
```

Comment:
this hash probably needs to be merged with another hash, something like `hash(session.summary_samples) + hash(session.interface)`

Reply:
otherwise interface changes wouldn't be captured (unless, is interface included in summary_samples?)

Reply:
@seveibar yes interface included in summary_samples?
maybe the naming is confusing, should I change it?

Reply:
summary_samples will contain this object
`{
  "interface": { /* ... */ },
  "summary": {
    "samples": [
      { "state": "complete", version: 32 }
      // ...
    ]
  }
}`

Reply:
gotcha, yea consider a rename in a future PR. "summary_object" is probably better